src/Service/AutoWiring/Resolver/PluginManagerResolver.php

```diff
@@ -20,23 +27,23 @@ class PluginManagerResolver implements ResolverInterface
      * @var array<class-string, string>
      */
     protected static array $pluginManagerMapping = [
-        'Laminas\Hydrator\HydratorInterface' => 'HydratorManager',
-        'Laminas\View\Helper\HelperInterface' => 'ViewHelperManager',
-        'Laminas\Validator\ValidatorInterface' => 'ValidatorManager',
-        'Laminas\Filter\FilterInterface' => 'FilterManager',
-        'Laminas\InputFilter\InputFilterInterface' => 'InputFilterManager',
-        'Laminas\InputFilter\InputInterface' => 'InputFilterManager',
-        'Laminas\Form\ElementInterface' => 'FormElementManager',
+        HydratorInterface::class => 'HydratorManager',
```

Comment:
This was referenced only by strings so you do not need to reference the package in the composer.json. 

Now you would need to do that. So I would suggest to let this be strings.